i18n/th_TH.csv

```diff
@@ -8,3 +8,5 @@
 "Choose your installment terms","เลือกระยะเวลาการผ่อนชำระ"
 "Miminum order value is 3000 THB", "ยอดสั่งซื้อขั้นต่ำ 3000 บาท"
 "Manage your cards", "จัดการบัตร"
+"more info", "โ้อมูลเพิ่มเติม"
+"Test mode", "โหมดทดลอง"
```

Comment:
maybe `โหมดทดสอบ` is better (saw Omise document uses `โหมดทดสอบ` instead of `โหมดทดลอง`)